_pages/changelog.md

```diff
@@ -4,6 +4,46 @@ permalink: /changelog/
 sidebar:
 nav: "changelog"
 ---
+# Version 7.0.15
+
+- Clarified that `FORM`.`MEDI` describe the original medium, not the derived medium, when used with derived files.
+
+- Clarified the meaning of the `WWW` structure, which previously only mentioned its payload datatype.
+
+- Clarified `PLAC` to both define "jurisdiction" and document its meaning in the absence of a `PLAC`.`FORM`.
+
+- Clarified what the term "principal date" means in different contexts in the definition of `g7:DATE`.
+
+- Updated `NICK` to no longer suggest that some names are "improper" and to document the diversity of views in what a "nickname" is.
+
+- Removed confusing reference to superstructures in the meaning of a documented extension tag.
+
+- Added ABNF for more datatypes and updated DIGIT's capitalization for compatibility with more ABNF toolchains.
+
+- Various typo corrections.
+
+
+# Version 7.0.14
+
+- Recommend that `NO XYZ` only be used where `XYZ` is permitted (its meaning is undefined elsewhere).
+
+- Recommend that a given `INDI` have at most one `FAMC` pointing to a given `FAM` (having more than one has unclear meaning); and likewise that a given `FAM` have at most one `CHIL` pointing to a given `INDI` (having more than one indicates nonsensical birth order).
+
+- Refactor presentation of local files to better match related RFCs and only make implementable constraints, and to use its own `<FilePath>` datatype instead of `<Special>`. This does not change `FILE` payloads, only how they are specified to better support automated tooling.
+
+- Refactor the enumeration tags `CENS`, `EVEN`, `FACT`, `NCHI`, and `RESI` to have different URIs, removing a previous parsing ambiguity. This changes neither the set of tags permitted in any enumeration set nor those tags' meaning, only how they are specified to better support automated tooling.
+
+- Deprecate extension-defined substructures using `stdTag` in a way incompatible with any standard definition of that tag. The now-deprecated use was common in 5.5.1 and is permitted in 7.0, but can prevent extension structures from being adopted as-is as new standard structures in future versions of the specification.
```

Comment:
```suggestion
- Deprecate the ability to use extension-defined substructures `stdTag` in a way incompatible with any standard definition of that tag. The now-deprecated use was common in 5.5.1 and is permitted in 7.0, but can prevent extension structures from being adopted as-is as new standard structures in future versions of the specification.
```
Possible wordsmithing suggestion to make it clearer that "in a way incompatible" modifies the use rather than the deprecation process.  Alternatively just change "using" to "that use".

Reply:
This suggestion is tracked in separate issue https://github.com/FamilySearch/GEDCOM/pull/547